### ISSUE TITLE
Getting text dif with line numbers

### ISSUE THREAD

**Issue by jkavalec** [kpdecker/jsdiff#519]
I would like do display line numbers before each line I output, is there any way to achieve this within library, without making custom implementation of this behavior?

**Comment by ThomasHambach** [comment]
You probably want to use this with another library. Somethin like https://diff2html.xyz/

**Comment by ExplodingCabbage** [comment]
Yeah there isn't really anything built into jsdiff for *visualising* the diffs.
